pkg/api/orchestrators.go

```diff
@@ -272,7 +266,7 @@ func dcosUpgrades(csOrch *OrchestratorProfile) ([]*OrchestratorProfile, error) {
 			OrchestratorVersion: common.DCOSVersion1Dot11Dot2,
 		})
 	}
-	return ret, nil
+	return ret
```

Comment:
This always returns `nil` for the error, so I removed that from the function signature.